extensions/blocks/instagram-gallery/edit.js

```diff
@@ -243,6 +253,15 @@ const InstagramGalleryEdit = props => {
 							min={ 0 }
 							max={ 50 }
 						/>
+						<ToggleControl
+							label={ __( 'Stack on mobile' ) }
```

Comment:
You'll want to add a textdomain here I think:

```suggestion
							label={ __( 'Stack on mobile', 'jetpack' ) }
```